tests/integration/targets/ec2_vol/tasks/main.yml

```diff
@@ -9,6 +9,7 @@
 
   collections:
     - amazon.aws
+    - community.aws
```

Comment:
```suggestion
```
Nit, we don't actually use community anywhere in these tests. Looks like this was added to the ec2_vol_info tests when we migrated erroneously.